grammar.js

```diff
@@ -211,7 +211,7 @@ module.exports = grammar({
         optional(
           seq(
             $.au_event_list,
-            alias(/[a-zA-Z*.]+/, $.pattern),
+            commaSep1(alias(/[a-zA-Z*.]+/, $.pattern)),
```

Comment:
should this be grouped in a node maybe? `(patterns(pattern+))`

Reply:
Yup, better in a node.

Reply:
Not sure in the end...

Reply:
It's fine like this I think.